uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -580,12 +580,6 @@ public function upgrade_1024()
   public function upgrade_1025() {
     $this->executeCustomDataFile('xml/activity_custom_fields.xml');
 
-    // set "remind_me" to true for all existing documents
-    civicrm_api3('Document', 'get', [
-      'options' => ['limit' => 0],
-      'api.Document.create' => ['id' => '$value.id', 'remind_me' => 1],
-    ]);
-
```

Comment:
It's because on staging there are some document for assignments. Each of these will create a revision because of the behaviour in the document API and activity API combined. When I checked for one of them after a few failed attempts to run the upgrade there were over 100 revisions created.

It fails because of a timeout when it tries to create so many revisions.